optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -213,9 +216,10 @@ def __call__(self, trial):
         if 'lambda_l2' in self.target_param_names:
             self.lgbm_params['lambda_l2'] = trial.suggest_loguniform('lambda_l2', 1e-8, 10.0)
         if 'num_leaves' in self.target_param_names:
-            max_depth = self.lgbm_params.get('max_depth', 8)
+            tree_depth = self.lgbm_params.get('max_depth', DEFAULT_TUNER_TREE_DEPTH)
+            max_num_leaves = 2 ** tree_depth if tree_depth > 0 else 2 ** DEFAULT_TUNER_TREE_DEPTH
```

Comment:
Could you remove the whitespaces around the power operators?

```suggestion
            max_num_leaves = 2**tree_depth if tree_depth > 0 else 2**DEFAULT_TUNER_TREE_DEPTH
```

This is not detected by autpep8, but [the official Python document](https://docs.python.org/3/reference/expressions.html#the-power-operator) uses this format.

And also, some files in Optuna employ this format as follows:
https://github.com/optuna/optuna/blob/master/optuna/samplers/tpe/sampler.py#L368

Reply:
Ops! Thanks for notifying me! Merged your suggestion.